lib/util/parse-cadence-events.ts

```diff
@@ -0,0 +1,27 @@
+import { CadenceEvent } from '@samatech/onflow-fcl-esm'
```

Comment:
App-specific and shouldn't be included

Reply:
I'm guessing you've meant `IFlowPurchaseEvent` here as well. I removed it.